### ISSUE TITLE
Customize URL

### ISSUE THREAD

**Issue by fjbardelli** [oscarmlage/django-cruds-adminlte#96]
It is possible to customize the URL, to give a complete form to the root, for example instead of

127.0.0.1:8000/app_two/model_1/list

to be

127.0.0.1:8000/api/list

Thank you so much

**Comment by oscarmlage** [comment]
I think so but Instead of defining the urls for the whole app like this:
```
urlpatterns += crud_for_app('testapp')
```
You should define the urls for every CRUD resource like this:

```
    url(r'/api/author', include(authorcrud.get_urls())),
    url(r'/api/invoice', include(invoicecrud.get_urls())),
    url(r'/api/customer', include(customercrud.get_urls())),
```
And you will get urls like:
* api/author/list
* api/author/create
* ...

Please take a look to the documentation and the testapp sample in the repo:

* https://django-cruds-adminlte.readthedocs.io/en/latest/installation.html#urls-for-the-crud
* https://github.com/oscarmlage/django-cruds-adminlte/blob/master/demo/demo/urls.py#L36

Dunno if it fits for you, feedback is always welcome ;)

**Comment by oscarmlage** [comment]
Maybe a good idea is be able to pass the url you want to `get_urls()` method to customize not only at model level but at resource level too. i.e.:

```
url(r'/my/custom/list', include(authorcrud.get_urls('list'))),
url(r'/my/custom/del', include(authorcrud.get_urls('delete'))),
```

What do you think guys? @luisza @migue56 @mbourqui

**Comment by fjbardelli** [comment]
this solution does not work, gives the following error and the server does not sell

path('api/ina/', include(MotivoInasistenciaCRUDView().get_urls('list'))),

    path('api/ina/', include(MotivoInasistenciaCRUDView().get_urls('list'))),
TypeError: get_urls() takes 1 positional argument but 2 were given

in this other

path('api/ina/', include(MotivoInasistenciaCRUDView().get_urls())),

I form this url

api/ina/

**Comment by oscarmlage** [comment]
> this solution does not work, gives the following error and the server does not sell

No, I don't mean it's working now like that, I mean that could be a good idea to extend the functionality -as a new feature- if you call `get_urls()` with an extra argument.

**Comment by fjbardelli** [comment]
Ok, sorry yes that great

Thanks

**Comment by mbourqui** [comment]
From a conceptual point of view, I think it makes sense, as you may not want to expose the full CRUD urls. Or as mentioned, customize only one or the other resource.
But I haven't check how feasible it would be.

**Comment by Seykotron** [comment]
I think that it could be nice to customize the URL and the URL name we can achieve this by making a dict attribute.

If dict is None then the "normal" behavior, else if the dict have the key "list" : { "name" = "auth_group_list", "url" : "/auth/group/list" }

It would be nice and easy I think :)

**Comment by chavanmahi** [comment]
![s](https://user-images.githubusercontent.com/49483930/64258844-928e4080-cf45-11e9-8503-e71a6a0a9c43.png)


I'm just getting this page it's something wrong in URL can u plz solve this
